frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown"></TextBlock>
+        <div class="flex justify-between mt-4">
+            <span>Current global limit</span>
+            <span v-if="liquidationLimits"
+                ><Explain :text="liquidationLimits.maximumProtocolDebtDai.toFixed()"></Explain> -
```

Comment:
Please don't use `toFixed` (which will display all possible decimal places) but the appropriate `Format` component 

(here and everywhere where you use `toFixed`)